gravitee-am-gateway/gravitee-am-gateway-standalone/gravitee-am-gateway-standalone-distribution/src/main/resources/templates/i18n/messages_fr.properties

```
@@ -23,3 +31,128 @@ reset_password.email.header.title=Bonjour {0}, une demande de r
+
+forgot_password.title=Changez votre mot de passe
+forgot_password.description=Nous vous enverrons des instructions de r�initialisation
+forgot_password.email.placeholder=E-mail
+forgot_password.button.submit=Envoyer
+forgot_password.success.title=V�rifiez votre messagerie
+forgot_password.success.description=Nous avons envoy� un lien de r�initialisation du mot de passe
+forgot_password.error.title=Erreur de mot de passe oubli�
+forgot_password.error.description=Veuillez revenir � votre application client et r�essayer, ou contacter le propri�taire et demander de l'aide
+forgot_password.error.description.label=Erreur de description :
+
+registration.title=Inscrivez-vous �
+registration.description=Vous avez d�j� un compte?
+registration.sign.in=Connexion
+registration.button.submit=S'inscrire
+registration.first.name.placeholder=Pr�nom
+registration.last.name.placeholder=Nom de famille
+registration.user.name=Nom d'utilisateur
+registration.email.placeholder=Courriel
+registration.password.placeholder=Mot de passe
+registration.success.title=Enregistr� avec succ�s
+registration.success.description=Merci d'avoir cr�� un compte
+registration.error.title=L'enregistrement a �chou�
+registration.error.description=Veuillez revenir � votre application client et r�essayer, ou contacter le propri�taire et demander de l'aide
+registration.error.description.label=Description de l'erreur:
+
+password.validation.label=Le mot de passe doit contenir:
+password.minLength.before=Contient au moins
+password.minLength.after=caract�res
+password.include.numbers=Contient un nombre
+password.include.special.characters=Contient un caract�re sp�cial
+password.letters.mixed.cases=Contient des lettres en casse mixte
+password.max.consecutive.letters.before=Max
+password.max.consecutive.letters.after=Lettres ou chiffres cons�cutifs
+password.exclude.common.passwords=N'utilisez pas de noms ou de mots de passe communs
+password.exclude.user.info=N'utilisez pas vos informations de profil dans le mot de passe
+
+webauthn.register.title=Authentification sans mot de passe
+webauthn.register.description=Suivez les instructions de l'�tape suivante pour vous connecter sans mot de passe. Votre appareil vous proposera diff�rentes options comme une cl� de s�curit�, un lecteur d'empreintes digitales, la reconnaissance faciale...
+webauthn.register.button.submit=Suivant
+webauthn.register.skip=Ignorer cette �tape
+webauthn.error=Utilisateur invalide
+
+webauthn.login.description=Utilisation de l'empreinte digitale, de l'appareil ou de la cl� de s�curit�
+webauthn.login.button.next=Suivant
+webauthn.login.tips=Suivez les instructions dans la fen�tre contextuelle de s�curit�
+webauthn.login.error=�chec de la connexion
+webauthn.login.error.description=Utilisateur invalide
+
+mfa.otp=Mot de passe � usage unique
+mfa.sms=SMS
+mfa.email=Courriel
```

```suggestion
mfa.email=Courriel
```
I think it is ok to keep "email"